tests/testthat/helper-ckanr.R

```diff
@@ -1,25 +1,72 @@
+#' Test whether the configured test CKAN is offline
+#'
+#' @keywords internal
+#' @importFrom testthat skip
+#' @details
+#' \code{check_ckan} will allow a test to fail (and not hide error messages)
+#' if \code{setup_ckanr(test_strict="TRUE")} was set.
+#' \code{check_ckan} will allow a test to skip (and the test suite to pass)
+#' if \code{setup_ckanr(test_strict)} was set to anything but the string "TRUE".
+#' @param url A URL that shall be tested whether it is online
+#'
```

Comment:
@florianm You shouldn't need any docstring for these test helper functions. You can do comments `#` if you want. AFAIK we shouldn't need to say we want to import `testthat::skip`

or do you mean for these functions to be available for use in the package via `:::`?